.github/workflows/integrate.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     - name: Install dependencies
       run: |
-        sudo apt-get install python3-setuptools
+        sudo apt-get install python3-pip
```

Comment:
Any reason to not use the actions/setup-python action instead like [here](https://github.com/canonical/bundle-istio/blob/master/.github/workflows/integration.yaml#L33)?

Reply:
We're not looking for a particular version of Python here, just whatever's in apt, so that seems like overkill over just using apt.